index.html

```diff
@@ -13,8 +13,14 @@
     <body>
         <header class="intro">
                 <a href="#" class="home_link">Daron Wiafe</a>
-            <nav class="portfolio_nav">
-                <i class="fas fa-bars fa-2x"></i>
+                <nav class="portfolio_nav">
+                    <ul>
+                    <li><i class="fas fa-bars fa-2x"></i></li>
+                    <li class="links">Portfolio</li>
+                    <li class="links">About</li>
+                    <li class="links">Contact</li>
```

Comment:
[OPTIONAL] It would be better to wrap these in `a` tags and link them with the relevant part of your page 😀

```
<li class="links"><a href="#<portfolio-section-id-here>">Portfolio</a></li>
```